packages/preload/src/index.ts

```diff
@@ -244,7 +244,7 @@ export function initExposure(): void {
     },
   );
 
-  contextBridge.exposeInMainWorld('navigateToRoute', async (routeId: string, args: unknown[]): Promise<void> => {
+  contextBridge.exposeInMainWorld('navigateToRoute', async (routeId: string, ...args: unknown[]): Promise<void> => {
```

Comment:
issue: this commit looks unrelated to the changes

Reply:
It is - unrelated fix found by Axel above. But since the fix is only 3 characters and this PR both removes the one old use of it (DeploymentColumnName) and adds the replacement (kube/column/Name) I was hoping I could get away with it 😀.

Reply:
I would issue just a small PR for that as it's a separate fix

also small PRs have a merge approval within minutes usually

Reply:
Ok: #11232